docs/ldml/tr35-personNames.md

```diff
@@ -780,19 +781,18 @@ Here are examples for Albert Einstein in Japanese and Chinese:
 #### 6.8.1 Setting the spaceReplacement
 
 1. The foreignSpaceReplacement is provided by the value for the `foreignSpaceReplacement` element; the default value is " ".
-
 2. The nativeSpaceReplacement is determined by the following algorithm, chosing between " " and "".
-
-1. Get the script of the formatting locale
-2. If the likely script is Thai, let nativeSpaceReplacement = " "
-3. Otherwise let nativeSpaceReplacement = "" if either of the following applies:
-    1. The script is Jpan, Hant, or Hans
-    2. The script has the script metadata property lbLetters = YES (this can also be algorithmically derived from the LineBreak property data).
-5. Otherwise, let nativeSpaceReplacement = ""
-
-3. If the formatter base language equals the name base language, then let spaceReplacement = nativeSpaceReplacement, otherwise let spaceReplacement = foreignSpaceReplacement.
+    1. Get the script of the formatting locale
+    2. If the likely script is Thai, let nativeSpaceReplacement = " "
+    3. Otherwise let nativeSpaceReplacement = "" if either of the following applies:
+        1. The script is Jpan, Hant, or Hans
+        2. The script has the script metadata property lbLetters = YES (this can also be algorithmically derived from the LineBreak property data).
+    4. Otherwise, let nativeSpaceReplacement = ""
```

Comment:
I'm pretty sure you mean "Otherwise, let nativeSpaceReplacement = `" "`.  As it stands, the nativeSpaceReplacement character is the empty string unless the script is Thai.